lib/views/public_body/_more_info.html.erb

```diff
@@ -17,6 +17,9 @@
     <% if tag_value.match(/^SC/) %>
       <%= link_to _('Charity registration'),
                     "https://www.oscr.org.uk/about-charities/search-the-register/charity-details?number=#{ tag_value }" %><br>
+    <% elsif tag_value.match(/^NIC/) %>
+      <%= link_to _('Charity registration'),
+                    "https://www.charitycommissionni.org.uk/charity-search/?searchText=#{ tag_value.gsub(/^NIC/, '') }" %><br>
```

Comment:
Was just about to merge this but then noticed we _could_, it seems, link direct to the Charity Details page.

e.g. for `charity:NIC102724`, we could link to https://www.charitycommissionni.org.uk/charity-details/?regId=102724 instead of https://www.charitycommissionni.org.uk/charity-search/?searchText=102724.

Any reason not to do this @mdeuk?

Reply:
```suggestion
                    "https://www.charitycommissionni.org.uk/charity-search/?regId=#{ tag_value.gsub(/^NIC/, '') }" %><br>
```

**I can't see any reason not to do this!**

I've ran through the database for NI and can't see any reason why this _**wouldn't**_ work. We didn't do it for CCEW as the identifiers in their database are awkward to translate (e.g. mySociety's reg of  1076346 becomes [3078648](https://register-of-charities.charitycommission.gov.uk/charity-search/-/charity-details/3078648)).

Good catch 👍